lib/delayed_job_groups_plugin.rb

```diff
@@ -11,6 +11,15 @@
 require 'delayed/job_groups/yaml_loader'
 require 'delayed/job_groups/version'
 
-Delayed::Backend::ActiveRecord::Job.send(:include, Delayed::JobGroups::JobExtensions)
+if defined?(Delayed::Backend::ActiveRecord)
```

Comment:
The gem has a dependency on delayed_job_active_record so won't the `Delayed::Backend::ActiveRecord` constant always be defined? Seems like this conditional could mask other issues and cause the gem to silently not load properly.

Reply:
You are very likely correct. I was following the pattern from the delayed_cron gem and didn't investigate why they required that check. Seems having it fail at load makes more sense than having it fail later when an attempt is made to use the group function. I will remove that if statement wrapper.